test/unit/Util/AstHelperTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflectionTest\Util;
+
+use PhpParser\Node\Stmt;
+use PHPUnit\Framework\TestCase;
+use Roave\BetterReflection\Reflector\DefaultReflector;
+use Roave\BetterReflection\SourceLocator\Type\StringSourceLocator;
+use Roave\BetterReflectionTest\BetterReflectionSingleton;
+use Roave\BetterReflection\SourceLocator\Ast\Locator;
+use Roave\BetterReflection\Util\AstHelper;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\Function_;
+use Roave\BetterReflection\Reflection\ReflectionFunction;
+
+
+
+final class AstHelperTest extends TestCase
```

Comment:
Let's add test coverage declarations here: we don't want coverage from this test to leak on other classes